internal/manager/metadata/metadata.go

```diff
@@ -3,8 +3,16 @@ package metadata
 
 import (
 	"strings"
+
+	"github.com/Kong/sdk-konnect-go/pkg/metadata"
 )
 
+func init() {
+	// NOTE: We do it this way because speakeasy does not provide a way to set the
+	// user-agent for the SDK instance.
+	metadata.SetUserAgent(UserAgent())
```

Comment:
The imported package name `metadata` looks confusing in another package named `metadata`. How about renaming the `github.com/Kong/sdk-konnect-go/pkg/metadata` to `sdkkonnectmetadata`?

Reply:
👍 

added in cf3274c710af547ed0d9025572f9ccf7c90b6186